executor/scripts/test_local_executor.py

```
@@ -0,0 +1,256 @@
+                print(f"  ✗ Status check returned {response.status_code}")
+                return {"connected": False, "error": response.text}
+        except Exception as e:
+            print(f"  ✗ Error checking executor status: {e}")
+            return {"connected": False, "error": str(e)}
+
+
+async def dispatch_test_task(
+    backend_url: str, prompt: str, anthropic_api_key: str = None
+) -> dict:
+    """Dispatch a test task to the local executor."""
+    print(f"\n[3/4] Dispatching test task...")
```

Ruff (0.14.14)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)